src/components/VueTable/VueTable.ts

```diff
@@ -922,6 +935,12 @@ export default class VueTable extends Vue {
           keyEnd: header,
         };
       }
+
+      this.highlight.tbody = [];
```

Comment:
why you don't call your function there ?
Call your function at the end of the function and add a comment
thx